db/schema.rb

```diff
@@ -28,6 +28,15 @@
     t.index ["neighbourhood_id"], name: "index_addresses_on_neighbourhood_id"
   end
 
+  create_table "articles", force: :cascade do |t|
+    t.text "title"
+    t.text "description"
+    t.date "published"
```

Comment:
should prob be "published_at" for consistency. maybe lets make this a DateTime too just in case we need the extra accuracy later?